### DISCUSSION TITLE
How do i search and replace entire project using fzf-lua without using quickfix list.

### DISCUSSION THREAD

**Discussion by 231tr0n** [ibhagwan/fzf-lua#1424] [Q&A] [answered]
Hi everyone. Is it possible to search and replace a keyword in selected locations in the entire project using fzf-lua. If so how can I do it.

**Comment by ibhagwan** [comment, marked as answer]
You’ll need to write a custom action in lua for that and use the regex (opts.search) or query to replace text at the selected lines.

**Reply by 231tr0n** [reply]
If I am not asking too much of you, can you please help me out by guiding me with atleast a basic implementation on which I can improve upon. It would be really helpful for me.

**Reply by 231tr0n** [reply]
I also had one more doubt. Does this kind of functionality makes sense to be implemented in fzf-lua itself.

**Reply by ibhagwan** [reply]
Read the wiki, start writing your custom action, use the neovim lua API whenever possible, use `FzfLua helptags` to understand more, when you have something you’re confused about post it here and I’ll help.

**Reply by ibhagwan** [reply]
> I also had one more doubt. Does this kind of functionality makes sense to be implemented in fzf-lua itself.

Nope, it’s already covered via many scenarios using the quick fix and local lists, it makes no sense for me to add it, `:cdo`, `:cfdo` should cover everything along with `:FzfLua grep_quickfix`.
